### PR TITLE
De-shortcode Services section.

### PR DESCRIPTION
**Pull request by chadwcarlson**
## Why

This PR removes most shortcodes from the "Add services" section of the documentation, for both Platform.sh _and_ Upsun, to make things easier to edit, understand, and eventually migrate.

More to come.

### CODE REVIEW

**Comment by chadwcarlson** [comment]
VRT tests to verify changes:

```
$ git clone git@github.com:gilzow/vrt-docs-migration.git
$ cd vrt-docs-migration
$ ./run-tests.sh
PR environment URL for docs.platform.sh? https://deshortcode-ybbt44i-ucq44jg6ofare.eu-5.platformsh.site/
What is the base path we are updating? add-services
Have you updated the template-paths.js file in the tests directory? Y/n: Y
```

File: sites/platform/src/add-services/elasticsearch.md [hunk 1]
```diff
@@ -101,12 +111,79 @@ To do so, follow the same procedure as for [upgrading](#upgrading).
...
-For [premium versions](#supported-versions),
-the service type is `elasticsearch-enterprise`.
+For [premium versions](#supported-versions), the service type is `elasticsearch-enterprise`.
 
 ## Usage example
 
-{{% endpoint-description type="elasticsearch" /%}}
+### 1. Configure the service
+
+To define the service, use the `elasticsearch`:
```
Comment (chadwcarlson) [inline sites/platform/src/add-services/elasticsearch.md:120]:
```suggestion
To define the service, use the `elasticsearch` type:
```
```suggestion
To define the service, use the `elasticsearch`:
```

Comment (chadwcarlson) [reply sites/platform/src/add-services/elasticsearch.md]: Updated this on all relevant pages in next commit.

**Comment by chadwcarlson** [comment]
@AnouckColson approved on my end 👍

**Review by AnouckColson — APPROVED** [review]

**Pull request merged**